package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "@nitric/sdk",
   "description": "Nitric NodeJS client sdk",
-  "nitric": "v0.22.0",
+  "nitric": "v0.27.0-rc.4",
   "author": "Nitric <https://github.com/nitrictech>",
   "repository": "https://github.com/nitrictech/node-sdk",
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
+  "version": "0.0.1",
```

Comment:
I believe the version key is added as part of the release process

Reply:
```suggestion
```